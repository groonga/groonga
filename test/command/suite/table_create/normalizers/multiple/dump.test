table_create Substitutions TABLE_PAT_KEY ShortText
column_create Substitutions substituted COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenBigram \
  --normalizers 'NormalizerNFKC130("unify_kana", true), \
                 NormalizerTable("column", "Substitutions.substituted")'
dump
