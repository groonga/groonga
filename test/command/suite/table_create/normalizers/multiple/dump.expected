table_create Substitutions TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Substitutions substituted COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizers 'NormalizerNFKC130("unify_kana", true),                  NormalizerTable("column", "Substitutions.substituted")'
[[0,0.0,0.0],true]
dump
table_create Substitutions TABLE_PAT_KEY ShortText
column_create Substitutions substituted COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer "NormalizerNFKC130(\"unify_kana\", true), NormalizerTable(\"column\", \"Substitutions.substituted\")"
